sematic/ui/packages/common/src/pages/RunDetails/sourcecode/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { useRunDetailsSelectionContext } from "src/context/RunDetailsSelectionContext";
+import SourceCode from "src/pages/RunDetails/sourcecode/SourceCode";
+
+export default function SourceCodePanel() {
+
+    const { selectedRun } = useRunDetailsSelectionContext();
+
+    if (!selectedRun) {
+        return null;
```

Comment:
Shouldn't this be an empty element?
```suggestion
        return </>;
```

Reply:
I think null is more economical than `<></>` (basically not even a virtual dom is adopted). Neither of them renders anything, and are almost equivalent.